src/main/java/com/flowpowered/nbt/ByteArrayTag.java

```diff
@@ -50,6 +50,11 @@ public ByteArrayTag(String name, byte[] value) {
         return value;
     }
 
+    @Override
+    public void setValue(byte[] value) {
```

Comment:
Why did you make every Tag mutable!? D': *cries*
I don't care what @kashike says, this is one of my favorite NBT libraries.
Immutable Tags are one reason for that.

If everyone really wants this.... *cries some more*
Maybe it should be in its own PR?

Reply:
Have you ever tried to change the position of an entity in chunk with the NBT data being immutable?

> Maybe it should be in its own PR?

IIRC this was, but I merged it onto my dev branch and I later build the API changes on top of it; sorry.